applications/SystemIdentificationApplication/python_scripts/controls/data_values_control.py

```diff
@@ -144,7 +145,10 @@ def __init__(self, name: str, model: Kratos.Model, parameters: Kratos.Parameters
         self.adjoint_model_part: Optional[Kratos.ModelPart] = None
 
         control_variable_bounds = parameters["control_variable_bounds"].GetVector()
-        self.clamper = self.container_type_helper.clamper_type(control_variable_bounds[0], control_variable_bounds[1])
+
+        # use the clamper in the unit interval
+        self.interval_bounder = IntervalBounder(control_variable_bounds)
```

Comment:
![image](https://github.com/user-attachments/assets/7c181b24-c5b3-40f2-988c-1611ac004416)

I don't think this is what you meant :laughing: please use proper English

Reply:
hmm... there are few applications in mathematics with the word ["Bounder"](https://math.stackexchange.com/questions/3636346/let-f-be-bounder-linear-functional-on-hilbert-space-h-then-dimensional-of). In any case, I don't mind changing it as well. So went ahead with the `ExpressionBoundingMaanger`]. Or I am open for suggestions :)